static/js/ui.js

```diff
@@ -46,6 +46,16 @@ function update_message_in_all_views(message_id, callback) {
     });
 }
 
+exports.show_error_for_unsupported_platform = function () {
+    if (/ZulipDesktop/.test(navigator.userAgent)) {
```

Comment:
@timabbott, @brockwhittaker FYI, output of `navigator.userAgent` is - 
```
"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/603.3.8 (KHTML, like Gecko) Zulip Desktop/0.5.2"
``` 
(there is space between **Zulip** **Desktop**)

Reply:
Has this been fixed, if not I can submit a PR for it.

Reply:
Go for it!